descriptor.md

```diff
@@ -176,6 +181,21 @@ In the following example, the descriptor indicates that the referenced manifest
 }
 ```
 
+In the following example, the descriptor indicates that the referenced manifest is retrievable from a particular URL:
+
+```json
+{
+  "mediaType": "application/vnd.example.signature+json",
+  "size": 3514,
+  "digest": "sha256:19387f68117dbe07daeef0d99e018f7bbf7a660158d24949ea47bc12a3e4ba17",
```

Comment:
this is the kind of place where multicodecs are useful https://github.com/multiformats/multicodec/blob/master/table.csv#L9